.github/workflows/osx.yml

```diff
@@ -0,0 +1,38 @@
+name: github-OSX
+
+on: [push, pull_request]
+
+jobs:
+  osxci:
+    name: osx-ci
+    runs-on: [macos-latest]
+
+    strategy:
+      matrix:
+        include:
+          - BACKEND: "SERIAL"
+            BUILD_TYPE: "RelWithDebInfo"
+          - BACKEND: "PTHREAD"
+            BUILD_TYPE: "RelWithDebInfo"
+          - BACKEND: "SERIAL"
+            BUILD_TYPE: "Debug"
+          - BACKEND: "SERIAL"
+            BUILD_TYPE: "Release"
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: configure
+        run:
+          cmake -B build \
+            -DKokkos_ENABLE_${{ matrix.BACKEND }}=On \
```

Comment:
Isn't SERIAL always on?

Reply:
It's on if no other host execution space is selected.

Reply:
Ok